src_base/xeve_bsw.c

```diff
@@ -240,8 +240,8 @@ void xeve_bsw_write_ue(XEVE_BSW * bs, u32 val)
     xeve_bsw_write(bs, code, len_c);
 }
 
-void xeve_bsw_write_se(XEVE_BSW * bs, int val)
+void xeve_bsw_write_se(XEVE_BSW * bs, u32 val)
```

Comment:
This function _se works on signed data. Function above _ue works on unsigned.
Your change is invalid.